inc/plugins/mybbfancybox.php

```diff
@@ -394,6 +394,25 @@ function mybbfancybox_showthread_start()
 	// Apply required changes in postbit_attachments_images_image template (replace all content)
 	$templates->cache['postbit_attachments_images_image'] = '<a target="_blank" data-fancybox="data-{$attachment[\'pid\']}" data-type="image"><img src="attachment.php?aid={$attachment[\'aid\']}" class="attachment" alt="" title="{$lang->postbit_attachment_filename} {$attachment[\'filename\']}&#13{$lang->postbit_attachment_size} {$attachment[\'filesize\']}&#13{$lang->mybbfancybox_uploaded} {$attachdate}&#13{$lang->mybbfancybox_views} {$attachment[\'downloads\']}{$lang->mybbfancybox_views_symbol_after}" /></a>&nbsp;&nbsp;&nbsp;';
 
+	// Minimize button - load JS code only when enabled in ACP
+	$minimize = '';
```

Comment:
You are setting `$minimize` here, but you haven't included it in the output HTML. Need to add `{$minimize}` to line 500:

```php
	});{$minimize}
	// -->
	</script>
```

Reply:
Added, thank you!